### PR TITLE
Mslq 77 (Assembly Guide)

### PR DESCRIPTION
**Pull request by harukins**
- Updated links to websites in the build materials table. Some of the parts are no longer available due to product discontinuation.
- Created Assembly Guide.

### CODE REVIEW

File: docs/source/hardware/build_materials.rst [hunk 1]
```diff
@@ -67,4 +67,18 @@ DC Power Plug         `Amazon <https://www.amazon.com/SIM-NAT-Pigtails-Security-
...
+#. Solder two 16 Gauge wires (red and black) to the bottom plate of F330, on top of where the 10 Gauge wires are soldered. The other end of the wires are soldered to　the input+/- terminals of the power distribution board for PixFalcon.
+#. Solder two 16 Gauge wires (red and black) to the bottom plate of F330, on top of where other wirse are soldered. The other end of the wires go to the input terminals of the Polulu Voltage Regulator.
+#. Solder a DC power plug to the output terminals of the voltage regulator. This plug will power the Odroid. 
+#. Attach the Pixfalcon power distribution board, Pololu voltage regulator, and the Odroid onto a laser-cut acrylic board. The Odroid is mounted with four 3D-printed Odroid Raiser (i.e. short standoffs) facing downwards. Attach the acrylic board to the F330 bottom plate.
+#. Attach two 3D-printed Landing Gears to the F330 bottom plate so the Odroid does not touch the ground directly.
+#. Sandwitch the four F330 arms with the bottom and the top plate of the F330 frame, using four screws only. The other four halls are for the 20mm male-female standoffs.
```
Comment (cacadam89) [inline docs/source/hardware/build_materials.rst]: Sandwitch --> Sandwich

File: docs/source/hardware/build_materials.rst [hunk 2]
```diff
@@ -67,4 +67,18 @@ DC Power Plug         `Amazon <https://www.amazon.com/SIM-NAT-Pigtails-Security-
...
+#. Attach a 3D-printed Pixfalcon Holder onto the top plate and tighten the screws.
+#. Stick the Pixfalcon flight controller carefully onto the Pixfalcon Holder with a piece of glue sponge that is included in the flight controller kit.
+#. Put a 3D-printed Pixfalcon Cover onto the four standoffs so it covers the flight controller.
+#. Use another set of four 20mm male-female standoffs to sandwitch the Pixfalcon Cover.
```
Comment (cacadam89) [inline docs/source/hardware/build_materials.rst]: sandwitch --> sandwich

File: docs/source/hardware/build_materials.rst [hunk 3]
```diff
@@ -67,4 +67,18 @@ DC Power Plug         `Amazon <https://www.amazon.com/SIM-NAT-Pigtails-Security-
 Assembly Guide
 ----------------------
 
-TBD
\ No newline at end of file
+#. Attach a MT2213 935KV motor to each F330 arm.
+#. Connect a ZTW Spider ESC to each motor. The ESCs should be physically attached to the arms with plastic tape so they do not interfere with propellers.
+#. Solder two 10 Gauge wires (red and black) to the bottom plate of F330, connecting the end of both wires to a XT 60 connector for battery connection.
+#. Solder two 16 Gauge wires (red and black) to the bottom plate of F330, on top of where the 10 Gauge wires are soldered. The other end of the wires are soldered to　the input+/- terminals of the power distribution board for PixFalcon.
+#. Solder two 16 Gauge wires (red and black) to the bottom plate of F330, on top of where other wirse are soldered. The other end of the wires go to the input terminals of the Polulu Voltage Regulator.
```
Comment (cacadam89) [inline docs/source/hardware/build_materials.rst]: wirse --> wires

File: docs/source/hardware/build_materials.rst [hunk 4]
```diff
@@ -35,17 +35,17 @@ The following parts are required per quadrotor:
...
-Motors: Crazepony 4pcs Emax MT2213 935KV                          1        `Amazon <https://www.amazon.com/gp/product/B00XY6T1W0>`_
-8045 Propeller Set (2xCW, 2xCCW>)                                 1        `Amazon <https://www.amazon.com/gp/product/B00T3VR85W/>`_
+PixFalcon Autopilot w/ GPS and Power Module                       1        `Amazon <https://www.amazon.com/Pixfalcon-M8N-GPS-Advanced-Bundle/dp/B017YCYVRO>`_
+F330 Frame                                                        1        `Amazon <https://www.amazon.com/Quadcopter-Frame-Aircraft-Accessory-Integrated/dp/B07D6K51DY>`_
+Motors: Crazepony 4pcs Emax MT2213 935KV                          1        `Amazon <https://www.amazon.com/MT2213-935KV-Brushless-Motor-Quadcopter-Multirotor/dp/B00N3I9GM4>`_
+8045 Propeller Set (2xCW, 2xCCW>)                                 1        `Amazon <https://www.amazon.com/DJI-Innovations-Quadcopter-Original-Propellers/dp/B00BS40S4A>`_
 ESC: ZTW Spider 30A OPTO 2-6S LiPo 600HZ Simonk OneShot125 ESC    4        `Amazon <https://www.amazon.com/gp/product/B01BXS7NQ6>`_
```
Comment (cacadam89) [inline docs/source/hardware/build_materials.rst:42]: this is one that is out of stock

File: docs/source/hardware/build_materials.rst [hunk 5]
```diff
@@ -35,17 +35,17 @@ The following parts are required per quadrotor:
...
 Spektrum 9645 DSMX Remote Receiver                                1        `Amazon <https://www.amazon.com/gp/product/B004M12GY6/>`_
-Turnigy 3300mah 3s 30c LiPo Battery                               1        `Amazon <https://www.amazon.com/gp/product/B00T3SULKO/>`_
+Turnigy 3300mah 3s 30c LiPo Battery                               1        `HobbyKing <https://hobbyking.com/en_us/turnigy-battery-3300mah-3s-30c-lipo-pack-xt-60.html>`_
 Spektrum DX6i 6 Channel Transmitter                               1        `Amazon <https://www.amazon.com/gp/product/B00K1P3KYW>`_
```
Comment (cacadam89) [inline docs/source/hardware/build_materials.rst:45]: this one is out of stock

also, the last item in this table is also out of stock on amazon (CP2102 Module USB 2.0 to TTL 6PIN). For some reason github wont let me add a comment to that line so I am sticking this here... (line 53)

File: docs/source/hardware/build_materials.rst [hunk 6]
```diff
@@ -35,17 +35,17 @@ The following parts are required per quadrotor:
...
-8045 Propeller Set (2xCW, 2xCCW>)                                 1        `Amazon <https://www.amazon.com/gp/product/B00T3VR85W/>`_
+PixFalcon Autopilot w/ GPS and Power Module                       1        `Amazon <https://www.amazon.com/Pixfalcon-M8N-GPS-Advanced-Bundle/dp/B017YCYVRO>`_
+F330 Frame                                                        1        `Amazon <https://www.amazon.com/Quadcopter-Frame-Aircraft-Accessory-Integrated/dp/B07D6K51DY>`_
+Motors: Crazepony 4pcs Emax MT2213 935KV                          1        `Amazon <https://www.amazon.com/MT2213-935KV-Brushless-Motor-Quadcopter-Multirotor/dp/B00N3I9GM4>`_
+8045 Propeller Set (2xCW, 2xCCW>)                                 1        `Amazon <https://www.amazon.com/DJI-Innovations-Quadcopter-Original-Propellers/dp/B00BS40S4A>`_
 ESC: ZTW Spider 30A OPTO 2-6S LiPo 600HZ Simonk OneShot125 ESC    4        `Amazon <https://www.amazon.com/gp/product/B01BXS7NQ6>`_
 Spektrum 9645 DSMX Remote Receiver                                1        `Amazon <https://www.amazon.com/gp/product/B004M12GY6/>`_
```
Comment (cacadam89) [inline docs/source/hardware/build_materials.rst:43]: this one is out of stock

**Review by cacadam89 — COMMENTED** [review]
@haruki-msl I know you mentioned in the PR that some items were out of stock, I wasnt sure if you meant to replace the links with in stock components or if you were just noting it. If it was the former, ignore my out-of-stock comments, but if its the later then some of the items are still out.

File: docs/source/hardware/build_materials.rst [hunk 7]
```diff
@@ -67,4 +67,18 @@ DC Power Plug         `Amazon <https://www.amazon.com/SIM-NAT-Pigtails-Security-
 Assembly Guide
 ----------------------
 
-TBD
\ No newline at end of file
+#. Attach a MT2213 935KV motor to each F330 arm.
+#. Connect a ZTW Spider ESC to each motor. The ESCs should be physically attached to the arms with plastic tape so they do not interfere with propellers.
+#. Solder two 10 Gauge wires (red and black) to the bottom plate of F330, connecting the end of both wires to a XT 60 connector for battery connection.
+#. Solder two 16 Gauge wires (red and black) to the bottom plate of F330, on top of where the 10 Gauge wires are soldered. The other end of the wires are soldered to　the input+/- terminals of the power distribution board for PixFalcon.
```
Comment (rhaksar) [inline docs/source/hardware/build_materials.rst]: there's a non-space character at "to the" in Step 4, in the sentence "The other end of the wires are soldered to the input+/-..."

File: docs/source/hardware/build_materials.rst (hunk 1 above)
Comment (harukins) [reply docs/source/hardware/build_materials.rst]: Fixed

File: docs/source/hardware/build_materials.rst (hunk 2 above)
Comment (harukins) [reply docs/source/hardware/build_materials.rst]: Fixed

File: docs/source/hardware/build_materials.rst (hunk 3 above)
Comment (harukins) [reply docs/source/hardware/build_materials.rst]: Fixed

File: docs/source/hardware/build_materials.rst (hunk 4 above)
Comment (harukins) [reply docs/source/hardware/build_materials.rst:42]: Looks like ESC, Receiver, and Transmitter are out-of-stock on many e-commerce sites. Leaving as they are based on our discussion last Monday.

File: docs/source/hardware/build_materials.rst (hunk 5 above)
Comment (harukins) [reply docs/source/hardware/build_materials.rst:45]: Fixed the link to CP2102 Module.

File: docs/source/hardware/build_materials.rst (hunk 6 above)
Comment (harukins) [reply docs/source/hardware/build_materials.rst:43]: Looks like ESC, Receiver, and Transmitter are out-of-stock on many e-commerce sites. Leaving as they are based on our discussion last Monday.

File: docs/source/hardware/build_materials.rst (hunk 7 above)
Comment (harukins) [reply docs/source/hardware/build_materials.rst]: Fixed.

**Pull request merged**